src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -96,20 +98,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new InvalidArgumentException(sprintf('Argument "name" not supported, it requires the Twig loader "%s".', FilesystemLoader::class));
         }
 
-        switch ($input->getOption('format')) {
-            case 'text':
-                $name ? $this->displayPathsText($io, $name) : $this->displayGeneralText($io, $filter);
-                break;
-            case 'json':
-                $name ? $this->displayPathsJson($io, $name) : $this->displayGeneralJson($io, $filter);
-                break;
-            default:
-                throw new InvalidArgumentException(sprintf('The format "%s" is not supported.', $input->getOption('format')));
+        if (\in_array($input->getOption('format'), $this->getAvailableFormatOptions())) {
+            $name ? $this->displayPathsText($io, $name) : $this->displayGeneralText($io, $filter);
+        } else {
+            throw new InvalidArgumentException(sprintf('The format "%s" is not supported.', $input->getOption('format')));
         }
 
         return 0;
     }
 
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('name')) {
+            $suggestions->suggestValues($this->getAvailableNameArguments($input->getArgument('name')));
+        }
+
+        if ($input->mustSuggestOptionValuesFor('filter')) {
```

Comment:
--filter allows to give any word, so it cannot be auto-completed.

Reply:
Removed.